Clients/Xamarin.Interactive.Client.Windows/ViewModels/WpfViewInspector.cs

```diff
@@ -0,0 +1,87 @@
+﻿//
+// Author:
+//   Larry Ewing <lewing@xamarin.com>
+//
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Windows;
+using Xamarin.Interactive.Logging;
+
+using Xamarin.Interactive.Client.Windows.Views;
+using Xamarin.Interactive.Client.Windows.Commands;
+using Xamarin.Interactive.Client.ViewInspector;
+
+namespace Xamarin.Interactive.Client.Windows.ViewModels
+{
+    sealed class WpfViewInspector<TView> :  ViewInspectorViewModel, IDisposable where TView : Window
```

Comment:
I don't really understand why this is generic? I only see it being instantiated for `AgentSessionWindow`.

Reply:
This is an artifact but it's pretty harmless.  If we get around to revisiting the highlighter it can be simplified then.